modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/Occurrence.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './Occurrence.scss';
+
+import React from 'react';
+
+import PanelNavigator from './PanelNavigator';
+
+import type {ImpactValue, Result} from 'axe-core';
+
+interface ICodeBlock extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactNode;
+}
+
+declare var Liferay: {
```

Comment:
We have some mocks [in-place](https://github.com/liferay/liferay-frontend-projects/blob/2d4996bb625532930b6a6812cfe997b04d83eeb8/projects/npm-tools/packages/npm-scripts/src/jest/mocks/Liferay.js), so we could add more if needed.

Theoretically, that's wrong, because `npm-scripts` is supposed to be used for any Portal version, thus the mocks could be different for 7.2, 7.3 and so on...

In practice, I think they don't differ as much as to cause any trouble, so I would say we may add them and see what happens. If one day we see it causes trouble, we may split the mocks by portal version and store them in `liferay-portal` instead of `npm-scripts` (or inside target platforms, where they could belong too).

Reply:
Yes, these mocks can be useful for setting up the types of our global functions :)

I think We will only get rid of this modules declarations when typing our `frontend-js-web` utils or create a file with the Liferay global functions typed on our [`tsconfig-base.json/`](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/config/tsconfig-base.json#L1) file on npm-scripts and adding this file globally